### PR TITLE
refactor: change thirdparty endpoint names

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

**Comment by lewisdaly** [comment]
@kleyow I feel like we should keep the names and enums both the same (`TP` instead of `THIRDPARTY`) for the sake of consistency.

**Comment by kleyow** [comment]
sounds good. I was bouncing on that myself. glad for the imput.

**Comment by lewisdaly** [comment]
Ooh we also need to bump the bugfix version in `package.json` @kleyow

**Comment by kleyow** [comment]
@lewisdaly should be good now.

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-services-shared",
-  "version": "11.2.0",
+  "version": "11.3.0",
```
Comment (lewisdaly) [inline package-lock.json:3]: This needs to match the `package.json` (you can just run an `npm install`)

Comment (kleyow) [reply package-lock.json]: done. I had bumped the minor then reverted to bump the patch instead without syncing.

**Review by lewisdaly — APPROVED** [review]

**Pull request merged**